1001-2000/LIT1295Dersan.xml

```diff
@@ -607,87 +499,58 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                </div>
                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamBeauties">
```

Comment:
Here you kept the new ID 'SalamBeauties' and dismissed the older reference to the month to indicate this salam.